src/CppUTestExt/MockSupport_c.cpp

```diff
@@ -72,28 +72,36 @@ static MockExpectedCall* expectedCall = NULL;
 static MockActualCall* actualCall = NULL;
 static MockFailureReporterForInCOnlyCode failureReporterForC;
 
-class MockCFunctionComparatorNode : public MockNamedValueComparator
+class MockCFunctionComparatorAndCopierNode : public MockNamedValueComparator, public MockNamedValueCopier
```

Comment:
I couldn't think of any. I tried eliminating the extra list, but couldn't
Look at it as these nodes implementing both interfaces. They're nice, clean interfaces and the node implements them. If this was Java, no one would think twice about it.
If you can think of a better solution, I'd be interested. Especially if we didn't need this list at all :)